src/components/Navbar/Navbar.js

```diff
@@ -4,7 +4,7 @@ import { NavLink } from 'react-router-dom';
 const Navbar = () => (
```

Comment:
- [ ] Great job so far 👏🏻 However the Styling must be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/). kindly Add the missing profile icon on the right side of the navigation bar 👇🏻 

![im1](https://user-images.githubusercontent.com/79058364/211124798-af2de6e5-c985-4c2a-97ce-1a345e26edfd.png)